Goobi/config/goobi_opac.xml

```diff
@@ -39,6 +39,9 @@
 			<searchField label="Barcode" value="8535" />
 			<searchField label="Barcode 8200" value="8200" />
 		</searchFields>
+		<resolve tag="028C" subtag="9">
```

Comment:
Additional comment: Your pull request example add a `searchField` attribute but this is missing or this attribute optional?